app/javascript/src/april-fools.js

```diff
@@ -25,6 +25,7 @@ async function hiimpetus(event) {
     source: element,
     friction: 0.96,
     boundX: [(window.innerWidth - element.offsetWidth - 100) * -0.5, (window.innerWidth - element.offsetWidth - 100) * 0.5],
+    boundY: [-element.offsetHeight, Infinity],
```

Comment:
Is this correct? Won't this cause it to not be able to go up all that far but still go off document down?

Reply:
Yes, that was my intention. If it goes below the document the scrollbar expands, so you can still scroll down and grab it again.

I could've added a lower bound as well, but I didn't feel like figuring out how to calculate the page's height with scrolling.

Reply:
I like the intention, but I think I'm ok with them going offscreen. Throwing them around aimlessly is fun!

Reply:
But then it's inconsistent: it can go out of the screen vertically, but bounces horizontally...
Isn't it more fun to not lose the damn thing if it goes off screen? That way you can keep playing with it 😜